### PR TITLE
Homework week 1 zagoksu

### PR DESCRIPTION
**Pull request by zagoksu**

### CODE REVIEW

File: Week1/homework/ex10.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+// Exercise 10: Compare arrays
+
+// Declare 2 variables, that each hold an array. The first array should have 4 items, the second 7 items
+// Find out how to get the length of each array. Write a console.log statement that shows the length of each array
+
+const arrayCountries = ["Germany", "Netherlands", "Belgium", "France"];
+
+console.log('The length of the array is' + ' ' + `${arrayCountries.length}`);
+
```
Comment (unmeshvrije) [inline Week1/homework/ex10.js]: too many unnecessary new lines

File: Week1/homework/ex2.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+'use strict'
+
+/*
+ console.log('I'm awesome');
+            ^^^
+SyntaxError: missing ) after argument list
```
Comment (unmeshvrije) [inline Week1/homework/ex2.js]: why was this error ? and why didn't you add a closing round bracket ) as the error said ?

File: Week1/homework/ex3.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+numberX = 7;
+
+// Next, add a console.log statement that explains what you think the value of numberX is.
+
+console.log("it should be 7 because it has been assigned value 0f 7.");
+
+// Add a console.log statement that logs the value of numberX. */
+
+console.log(numberX);
+
```
Comment (unmeshvrije) [inline Week1/homework/ex3.js]: good job, but too many new lines

File: Week1/homework/ex5.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+let a = Math.round(z);
+
+// Write a console.log statement in which you log the value of a.
+
+console.log(a);
+
+
+// So now we have z and a find a way to compare the two values and store the highest of the two in a new variable.
+
+let highestValue = Math.max(z, a);
```
Comment (unmeshvrije) [inline Week1/homework/ex5.js]: good that you found Math.max()

File: Week1/homework/ex8.js [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+    console.log("SAME TYPE");
+} else{
+    console.log("Not the same...");
+}
+
+if (typeof car == typeof house){
+    console.log("SAME TYPE");
+} else{
+    console.log("Not the same...");
+}
```
Comment (unmeshvrije) [inline Week1/homework/ex8.js]: well done!

File: Week1/homework/ex2.js (hunk 2 above)
Comment (zagoksu) [reply Week1/homework/ex2.js]: I've changed opening and closing quotes to double ones to solve it.

File: Week1/homework/ex3.js (hunk 3 above)
Comment (zagoksu) [reply Week1/homework/ex3.js]: i thought it would be more readable.

File: Week1/homework/ex10.js (hunk 1 above)
Comment (zagoksu) [reply Week1/homework/ex10.js]: i will leave no space next time.

**Comment by zagoksu** [comment]
> Good work. some minor changes about styling (newlines).

Thank you for the feedback.

**Pull request closed without merging**